pkgs/sdk/testutils/testutils.go

```diff
@@ -37,14 +37,15 @@ func (msg *TestMsg) GetSigners() []crypto.Address {
 
 //----------------------------------------
 // Utility Methods
-
+// Fee gas-wanted 5000, gas-fee 1gnot.
 func NewTestFee() std.Fee {
-	return std.NewFee(50000, std.NewCoin("atom", 150))
+	
```

Comment:
```suggestion	
```

Can you try to configure your IDE to run linting tools like gofumpt automatically?